requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 click~=8.0
 klat-connector~=0.6,>=0.6.2a21
```

Comment:
Update to stable 0.8 spec after https://github.com/NeonGeckoCom/klat-connector/pull/92